velox/common/memory/Memory.h

```diff
@@ -250,6 +254,7 @@ class MemoryManager {
   const bool checkUsageLeak_;
   const bool debugEnabled_;
   const bool coreOnAllocationFailureEnabled_;
+  const MemoryPoolImpl::CapacityGrowCallback poolGrowCb_;
```

Comment:
Can we have detailed comment on this field as it is a critical externally exposed API? Thanks